ksql-api/src/test/java/io/confluent/ksql/api/TlsTest.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2020 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.api;
+
+import io.confluent.ksql.api.server.ApiServerConfig;
+import io.confluent.ksql.test.util.secure.ServerKeyStore;
+import io.vertx.core.http.HttpVersion;
+import io.vertx.ext.web.client.WebClientOptions;
+import java.util.HashMap;
+import java.util.Map;
+import org.apache.kafka.common.config.SslConfigs;
+
+public class TlsTest extends ApiTest {
```

Comment:
Do we want to run _all_ the tests in `ApiTest` with Tls enabled?  I would of thought that's a bit heavy weight given the number of tests in there will likely increase over time.  Is it not sufficient to test that a single pull query works, a single push query works, etc?

Reply:
A caught a bug during development which only caused one of those tests to fail using TLS so I think there is value in running all of them. They run quite quickly currently and certainly compared to the integration tests run time is not significant, but if it becomes a problem we can revisit.